### DISCUSSION TITLE
Welcome to CODStatusBot Discussions!

### DISCUSSION THREAD

**Discussion by bradselph** [bradselph/CODStatusBot#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Cemo99** [comment]
An error occurred while checking this account. Please try again later.

getting this error for checking accounts i got cookie and api set correctly.

**Reply by bradselph** [reply]
Are you using your own installation or the officially approved version on Discord?

What provider are you using for the captcha solver?

You can run the `/accountage` command to test whether the bot is functioning properly. It's the most basic command it sends a request to Activision using only the account cookie and does not require a solver. If that command fails, there may be other underlying issues.

Feel free to contact me on Discord for faster responses. If you're interested in contributing or just want a better understanding of the project, I'm more active there than here.
